README.md

```diff
@@ -1,742 +1,77 @@
-# OpenTelemetry .NET SDK - distributed tracing and stats collection framework
-
-.NET Channel: [![Gitter chat][dotnet-gitter-image]][dotnet-gitter-url]
-
-Community Channel: [![Gitter chat][main-gitter-image]][main-gitter-url]
-
-We hold regular meetings. See details at [community page](https://github.com/open-telemetry/community#net-sdk).
-
-Approvers ([@open-telemetry/dotnet-approvers](https://github.com/orgs/open-telemetry/teams/dotnet-approvers)):
-
-- [Bruno Garcia](https://github.com/bruno-garcia), Sentry
-- [Christoph Neumueller](https://github.com/discostu105), Dynatrace
-- [Cijo Thomas](https://github.com/cijothomas), Microsoft
-- [Liudmila Molkova](https://github.com/lmolkova), Microsoft
-
-*Find more about the approver role in [community repository](https://github.com/open-telemetry/community/blob/master/community-membership.md#approver).*
-
-Maintainers ([@open-telemetry/dotnet-maintainers](https://github.com/orgs/open-telemetry/teams/dotnet-maintainers)):
-
-- [Mike Goldsmith](https://github.com/MikeGoldsmith), LightStep
-- [Sergey Kanzhelev](https://github.com/SergeyKanzhelev), Microsoft
-
-*Find more about the maintainer role in [community repository](https://github.com/open-telemetry/community/blob/master/community-membership.md#maintainer).*
-
+# OpenTelemetry .NET
+[![Gitter chat](https://img.shields.io/gitter/room/opentelemetry/opentelemetry-dotnet)](https://gitter.im/open-telemetry/opentelemetry-dotnet)

```

Comment:
mention auto-instr channel as well please

Reply:
I think we want to follow Java model and have a separate repo/SIG?
For example https://github.com/open-telemetry/opentelemetry-auto-instr-java.

Reply:
It doesn't exist yet as we need to decide how to proceed. I thought to mention this here for now

Reply:
Given it is not previously mentioned in the doc and we just get started on the auto-instrumentation, I suggest that we make decision on whether we should have a separate repo/SIG next week, and leave it outside this PR.